### ISSUE TITLE
direction of motion

### ISSUE THREAD

**Issue by 3073** [akshaychawla/Optical-Flow-Tracking----OpenCV#1]
can I tell direction of motion with this?

**Comment by akshaychawla** [comment]
Not exactly. Optical flow tracks the movements of point in 2d, however these points are actually moving in the 3d space. 
To get an estimate of the motion of the camera/object, you can obtain the features tracked in the variable "new_corners" on line 67 in tracking6.py and subtract them from "old_corners" to get an generic direction of motion per tracked feature. 
You could average this direction per tracked feature to get a global direction of motion. It won't be an exact measure though.

**Comment by 3073** [comment]
thank you.
